Backend_Application/src/main/java/dsd/cohort/application/ingredient/IngredientService.java

```diff
@@ -0,0 +1,14 @@
+package dsd.cohort.application.ingredient;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public interface IngredientService {
+
+    IngredientEntity getIngredientById(Long id);
+
+    IngredientEntity getIngredientByName(String name);
```

Comment:
The name field has a chance to not be unique, do you want to instead use the foodId value here?

Reply:
I think this depends if the food id comes in the recipes API data?

Reply:
That's a good point, I'll refer to the API response data and get back to you.